### PR TITLE
Create pull_request_template.md

### PR DESCRIPTION
**Pull request by siandreev**

### CODE REVIEW

File: pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+## Supported features
+- [ ] ton_proof
+- [x] SendTransaction
+
+
+## Tests
+[a demo dapp with integrated wallet](link)
+
+## Integrator contacts
+telegram/discord/email/...
```
Comment (oleganza) [inline pull_request_template.md:19]:
```suggestion
* telegram
* email
* discord
* ...
```

File: pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
+# Add <WALLET_NAME> wallet
+
+## Supports
+- [x] JS bridge as a browser extention for [Google Chrome](<chrome store url>), ..., browsers.
+- [ ] JS bridge as a mobile wallet app with own browser for [ios](<appstore link>) and [android](<google play link>).
+- [ ] JS bridge as a desctop wallet app with own browser for [windows](<link>), [mac](<link>), ... .
```
Comment (oleganza) [inline pull_request_template.md:6]:
```suggestion
- [ ] JS bridge for in-wallet browser for [Windows](<link>), [macOS](<link>), ... .
```

File: pull_request_template.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+- [x] HTTP bridge as a mobile wallet app for [ios](<appstore link>) and [android](<google play link>).
```
Comment (oleganza) [inline pull_request_template.md:7]:
```suggestion
- [x] HTTP bridge as a mobile wallet app for [iOS](<appstore link>) and [Android](<google play link>).
```

File: pull_request_template.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+- [ ] HTTP bridge as a desctop wallet app for [windows](<link>), [mac](<link>), ... .
```
Comment (oleganza) [inline pull_request_template.md:8]:
```suggestion
- [ ] HTTP bridge as a desctop wallet app for [Windows](<link>), [macOS](<link>), ... .
```

File: pull_request_template.md [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+
+## Supported features
+- [ ] ton_proof
```
Comment (oleganza) [inline pull_request_template.md:11]: We could add links to the spec/documentation for each item here so it's easier to reference them right away.

Comment (siandreev) [reply pull_request_template.md]: добавил

**Pull request merged**